package.json

```diff
@@ -35,8 +32,6 @@
   },
   "scripts": {
     "build": "svelte-package --input ./src && pnpm lint:package",
-    "prepublishOnly": "pnpm test && pnpm build",
-    "postpublish": "del ./dist",
```

Comment:
I was kind of curious why this was running, perhaps to ensure locally old artifacts are cleaned up.

Reply:
You don't really need to clean it up in CI for this workflow. It's going to run tests, build the latest, and then publish. it won't be pushing anything back to the repo, so the post-cleanup isn't necessary.

Reply:
Just saying if the folder isn't cleaned there could be left over files that get published.

Reply:
Either way, this doesn't work on windows / cross platform. I'm fine with removing it because I'd rather have github actions always do publishing

Reply:
Is there a reason to have

```yaml
      - name: Check size
        uses: andresz1/size-limit-action@master
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

instead of
```yaml
      - name: Check size
        run: pnpm test:size
```

Reply:
Yes

The main features of this action are:

- Commenting pull requests with the comparison of Size Limit output.
- Rejecting a pull request if the cost exceeds the limit.

Reply:
k let me try reverting this

Reply:
🤞🏻

Reply:
check size is trying to compare an outdated lockfile since it wasn't updated when the remove of `publishConfig` was pushed. What I don't understand is why it's not taking the latest from master.